seed/static/seed/js/controllers/inventory_list_beta_controller.js

```diff
@@ -1108,33 +1064,57 @@ angular.module('BE.seed.controller.inventory_list_beta', [])
 
       $scope.model_actions = 'none';
       elSelectActions = document.getElementById('select-actions');
-      $scope.run_action = function () {
+      $scope.run_action = function (viewIds = []) {
+        let selectedViewIds = [];
+
+        // was the function called with a list of ids?
+        if (viewIds.length > 0) {
+          selectedViewIds = viewIds;
+
+        // if it appears everything selected, only get the full set of ids...
+        } else if ($scope.selectedCount == $scope.inventory_pagination.total) {
+          selectedViewIds = [];
+
+          $http.post('/api/v3/properties/filter/', {}, {
+            params: {
```

Comment:
Should we worry about the current cycle here?

Reply:
That's probably not a bad idea, Let's add cycle here